packages/shared/src/lib/core/network/constants/supported-network-id-to-noves-chain.constant.ts

```diff
@@ -0,0 +1,13 @@
+import { NetworkId } from '../types'
+import { SupportedNetworkId } from './supported-network-id.constant'
+
+export const SUPPORTED_NETWORK_ID_TO_NOVES_CHAIN: Readonly<{ [key in NetworkId]?: string }> = {
+    [SupportedNetworkId.ShimmerEvm]: 'shimmer-evm',
```

Comment:
these values might change, so best to have the raw values within the noves modules so its easier to find if needed.
Then you can still have the map here or have the map in the same module.